src/help.cpp

```diff
@@ -269,6 +269,28 @@ void add_options() {
   opt->setSQL();
   opt->setDDL();
 
+  /* Create Undo Tablespace */
+  opt = newOption(Option::INT, Option::CREATE_UNDO_TABLESPACE, "undo_tbsp");
+  opt->help = "Create undo tablespaces";
```

Comment:
Explain help. It is the number of default undo table-spaces. Give it a name UNDO_TABLESPACES.